### DISCUSSION TITLE
How to impose "full developed flow" condition?

### DISCUSSION THREAD

**Discussion by HYB777** [multiphenics/multiphenicsx#30] [Q&A]
Hi, I want to impose the full developed flow condition on the inlet boundary, which means the average inlet velocity is given. The condition is formulated as follow:

$\frac{1}{-|\Gamma_{\text{inlet}}|}\int_{\Gamma_{\text{inlet}}}\mathbf{u}\cdot\mathbf{n}ds=u_{\text{avg}}$.

For my case, the domain is $[0, 1]\times[0, 1]$, and the inlet boundary is at $y=1$.

How should I do for this?

**Comment by francesco-ballarin** [comment]
Please do not duplicate questions here and on FEniCS discourse. It is the second time you do that in the last few days.
